include/klee/ExecutionState.h

```diff
@@ -173,6 +176,8 @@ class ExecutionState {
 
   bool merge(const ExecutionState &b);
   void dumpStack(llvm::raw_ostream &out) const;
+
+  std::string getNewMathVarName(std::string mathFunctionName);
```

Comment:
@Himeshi This we can move to `ErrorState` so that we create both the math and math error variables in the same place. I think this will make the code easier to read.